CMakeLists.txt

```diff
@@ -209,20 +209,33 @@ if( MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPOC")
     endif()
 endif()
 message( STATUS "${MIOPEN_BACKEND} backend selected." )
-# look for and register extractkernel
-find_program(EXTRACTKERNEL_BIN extractkernel
-    PATH_SUFFIXES bin
-    PATHS /opt/rocm
-)
 if(CMAKE_CXX_COMPILER MATCHES ".*hcc")
```

Comment:
Looks like this fails on CXX=clang++-3.8

Reply:
it may not be at /opt/rocm/bin
should check by this order
/opt/rocm/hip/bin
/opt/rocm/hcc/bin
/opt/rocm/bin

Reply:
Will replace `"clang\\+\\+"` with `"clang\\+\\+$"`